src/SmartComponents/Reports/Reports.js

```diff
@@ -19,19 +19,24 @@ import {
 
 const QUERY = gql`
 {
-    allProfiles {
-        id
-        name
-        refId
-        description
-        totalHostCount
-        compliantHostCount
-        majorOsVersion
-        complianceThreshold
-        businessObjective {
-            id
-            title
+    profiles(search: "has_test_results = true", limit: "1000"){
```

Comment:
Limit needs to be an integer

```suggestion
    profiles(search: "has_test_results = true", limit: 1000){
```